server/schedulers/utils.go

```diff
@@ -375,3 +379,18 @@ func (q *retryQuota) GC(keepStores []*core.StoreInfo) {
 		}
 	}
 }
+
+func newTestCluster(needToRunStream ...bool) (context.CancelFunc, *config.PersistOptions, *mockcluster.Cluster, *schedule.OperatorController) {
```

Comment:
Shall we move it to testutil?

Reply:
This function will remain in the schedulers pkg, as it depends on schedulers.register